### PR TITLE
CORE-195 - Laravel 5.7 Upgrade

### PR DESCRIPTION
**Pull request by CalumTowers**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
   },
   "devDependencies": {
     "admin-lte": "2.3.11",
-    "axios": "^0.16.2",
-    "bootstrap": "^3.3.7",
+    "axios": "^0.18",
+    "bootstrap": "^4.0.0",
```
Comment (AxonC) [inline package.json]: Ooh someone is brave... are we sure this is going to be as simple as a upgrade?

File: routes/web.php [hunk 2]
```diff
@@ -7,7 +7,3 @@
 Route::group(['domain' => config('app.url'), 'middleware' => ['admin']], function () {
     require base_path('routes/web-admin.php');
 });
-
-Route::group(['domain' => 'vats.im'], function () {
```
Comment (AxonC) [inline routes/web.php]: Why are we removing this?

File: package.json (hunk 1 above)
Comment (CalumTowers) [reply package.json]: Tested and appears to be working fine (Laravel now ships with 4.0.0 out of the box). Feel free to have a test!

Comment (atoff) [reply package.json]: +1 with the bootstrap upgrade, that will break quite a lot. Off the top of my head, panels are now cards to start with

File: routes/web.php (hunk 2 above)
Comment (CalumTowers) [reply routes/web.php]: No longer hosted by VATSIM UK or Core, now its own service at https://vats.im/ and https://github.com/NFarrington/vatsim-url-shortener

**Pull request merged**